### DISCUSSION TITLE
FieldSet.from_croco(): using `w` or `omega` for vertical velocity?

### DISCUSSION THREAD

**Discussion by erikvansebille** [Parcels-code/Parcels#1728] [General]
This discussion item is to discuss experiences with using either `w` or `omega` as vertical velocity in `FieldSet.from_croco()`. Please add comments below

**Comment by JamiePringle** [comment]
Erik -- this discussion on the ROMS forum may be helpful. https://www.myroms.org/forum/viewtopic.php?t=5532 Let me know if you can't read it. in short, as I imperfectly understand it, omega is with respect to the grid (as if it was not deformable), and w is the actual physical velocity.  But both should have the same units. Does this help? (p.s. CROCO people need to chime in if they changed things).
